benchmarks/README.md

```diff
@@ -0,0 +1,99 @@
+# ZK-Kit: Merkle Tree Benchmarks
+
+This project is part of the `zk-kit` repository and includes benchmarking tests for three Merkle Tree implementations:
+
+-   **Incremental Merkle Tree (IMT)** from `@zk-kit/imt`
+-   **Lean Incremental Merkle Tree (LeanIMT)** from `@zk-kit/lean-imt`
+-   **Sparse Merkle Tree (SMT)** from `@zk-kit/smt`
+
+The benchmark compares the performance of each Merkle Tree implementation for operations like adding leaves, generating proofs, and verifying proofs.
+
+## Table of Contents
+
+-   [Installation](#installation)
+-   [Usage](#usage)
+-   [Benchmark](#benchmark)
+-   [Results](#results)
+-   [Dependencies](#dependencies)
+
+---
+
+## Installation
+
+Before running the benchmark, ensure that you have Node.js version 20 installed and the required dependencies set up.
+
+1. **Clone the Repository**:
+    ```bash
+    git clone https://github.com/privacy-scaling-explorations/zk-kit.git
+    cd zk-kit
+    yarn
+    ```
+
+## Usage
+
+To run the benchmark, use the following command:
+
+```bash
+npx ts-node benchmarks/index.ts
```

Comment:
can't we mention the already defined `benchmarks` script here instead?
https://github.com/privacy-scaling-explorations/zk-kit/blob/2173bc800d2a22409678a7d3983df48d4f8efd18/package.json#L24

Reply:
Thanks, I have already changed it.